test/extended/images/db_helpers.go

```diff
@@ -0,0 +1,120 @@
+package images
```

Comment:
this helpers will be available for every test file in "images" directory. If we want to call them from a different place (for example testing some example application), you have to import the image tests which is not what you want, right?